client/blocks/app-banner/style.scss

```diff
@@ -91,10 +92,14 @@ body.app-banner-is-visible {
 
 	&.jetpack {
 		font-size: rem(28px); //typography-exception
-		font-weight: bold;
+		font-weight: 500;
```

Comment:
While I see the weight in Figma is "Medium," for my eyes, it seems like `bold` renders more closely to the design comp than `500`. It is quite nuanced and likely not that important either way. Just wanted to drop a note, though.